dependent-sum/dependent-sum.cabal

```diff
@@ -49,7 +49,7 @@ Library
                       , constraints-extras >= 0.2 && < 0.4
 
   -- tight bounds, so re-exported API is versioned properly.
-  build-depends:        some >= 1.0.0.3 && <1.0.1
+  build-depends:        some >= 1.0.0.3 && <1.0.2
```

Comment:
The lower bound have to be `>=1.0.1`, otherwise API could drift. I.e. include `withSomeM` or not by pure luck of what solver picks up!

Reply:
True. Thanks. I'm inclined to make a breaking change soon that will just stop re-exporting this. What do you think?

Reply:
It's up to you. I generally don't like re-exporting modules, so I'm sympathetic. OTOH, you just made 0.7 release, so maybe wait until something else would need a release.

I have nothing for `some`, e.g. Currently it looks to be "done".

Reply:
0.7 makes a good migration path to the non re-export version.